templates/cfp.html

```diff
@@ -1,7 +1,7 @@
 {% extends "body.html" %}
 {% block content %}
 <!-- Inner Page Banner Area Start Here -->
-<section class="inner-page-banner" style="background-image: url({{ url_for('static', filename='img/about/header1.jpg') }});">
+<section class="inner-page-banner" style="background-image: url({{ url_for('static', filename=cover) }});">
```

Comment:
Once I am doing this review I have realized that we shouldnt use inline style and use css class instead...

We should creaet a css class with background image, and than just pass the cover variable to template and the variable would be just css class. This way we dont need the `get_cover` at all.

We just pass the class name to load, good thing about css is that if the class doesnt exists browser will ignore it so no need to check if file exists in Python, we just make sure we have correct path in css.